sympy/tensor/indexed.py

```diff
@@ -134,7 +134,7 @@ class Indexed(Expr):
     """
     is_commutative = True
 
-    def __new__(cls, base, *args, **kw_args):
+    def __new__(cls, base, *args):
```

Comment:
It was not being used. Also if you try to pass a kwarg, ``Expr`` doesn't allow it. So, I removed it.